guides/v2.1/test/unit/writing_testable_code.md

```diff
@@ -151,6 +151,62 @@ Shorter methods do less, which in turn means they are easier to test. The same i
 
 As a rule of thumb, try to keep methods to five or fewer lines of code.
 
+### Each function has one function
```

Comment:
I would recommend rephrasing it as `Each function has one single purpose` to avoid tautology :)